examples/quickstart/variables.tf

```diff
@@ -26,7 +26,7 @@ variable "region" {
 }
 
 variable "ssh_key" {
-  description = "Public SSH key to use to provision a VSI. Must be a valid SSH key that does not already exist in the deployment region. See https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys."
+  description = "Public SSH Key for VSI creation. Must be an RSA key with a key size of either 2048 bits or 4096 bits (recommended) - See https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys. If key already exists in account, it will be used and no new key will be created."
```

Comment:
"If key already exists in account, it will be used and no new key will be created" -> this is not fully accurate. The limitation is per region. It is possible to have the same key in the same account across multiple region.